editor/components/block-list/block-draggable.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Draggable } from '@wordpress/components';
+
+function BlockDraggable( { rootUID, index, uid, layout, isDragging, ...props } ) {
```

Comment:
Minor: This component shouldn't need to rely on the parent to pass in `isDragging`, since this state can be known by the callbacks provided through its own `Draggable` element.

Reply:
That's true, but it's needed also on the parent so I thought about avoiding the duplicate logic.

Reply:
Fair enough. If it was a separate component not within the same folder, I think it'd be a bigger issue, but for these types of components I think it's okay-ish to make some assumptions about the behavior of the parent component.